### ISSUE TITLE
Question about some formula

### ISSUE THREAD

**Issue by Hugo-cell111** [BBBBchan/CorrMatch#3]
Hi! For the equation(4), it is written as KL(logits(weak), logits(strong)), but for the KL(P||Q), P usually represents the real distribution, while Q represents the optimal/ideal distribution, so I think do we need to change the order of two kinds of logits? Plus, In formula (9) first term, does the pseudo label of weak augmented branch serve as the pseudo supervision of the correlation map of its own? Thanks!

**Comment by BBBBchan** [comment]
Thanks! 
1. Sorry for the misleading. We actually do not change the order of the two kinds of logits. The KL loss is calculated from logits(strong) to logits(weak).
```python
            softmax_pred_u_w = F.softmax(pred_u_w.detach(), dim=1)
            logsoftmax_pred_u_s1 = F.log_softmax(pred_u_s1, dim=1)

            loss_u_kl_sa2wa = criterion_kl(logsoftmax_pred_u_s1, softmax_pred_u_w)
            loss_u_kl_sa2wa = torch.sum(loss_u_kl_sa2wa, dim=1) * conf_fliter_u_w
            loss_u_kl_sa2wa = torch.sum(loss_u_kl_sa2wa) / torch.sum(ignore_mask_cutmixed1 != 255).item()
```
2. The pseudo label of weak augmented branch  $p^w$  serves as the supervision of the correlation branch.
```python
            loss_u_corr_w = criterion_u(pred_u_w_corr, mask_u_w)
            loss_u_corr_w = loss_u_corr_w * ((conf_u_w >= thresh_global) & (ignore_mask != 255))
            loss_u_corr_w = torch.sum(loss_u_corr_w) / torch.sum(ignore_mask != 255).item()
```

You can refer to our code for more details.

**Comment by Hugo-cell111** [comment]
Hi! In CE Loss, the prediction is usually obtained by softmax. But I haven't found any softmax function applied to pred_u_w_corr in your code, which means pred_u_w_corr is model logit not the probability. So I feel it a little weird and wonder how you decide the loss function in this way. Thanks!
